fixtures/mockData.tsx

```diff
@@ -0,0 +1,44 @@
+const data = [
+  {
+    id: 1,
+    fullName: "Niene Boyen",
+    accesTo: "DEGO DOOK Autobranche",
```

Comment:
Try to use arrays when stacking data

```suggestion
    accessTo: [DEGO, DOOK, Autobranche],
```